samcli/commands/build/command.py

```diff
@@ -216,6 +245,8 @@ def do_cli(  # pylint: disable=too-many-locals, too-many-statements
     if use_container:
         LOG.info("Starting Build inside a container")
 
+    processed_env_vars = _process_env_var(container_env_vars)
```

Comment:
when customers do not provide any env vars, is `container_env_vars` `None` or empty list?

Reply:
It's going to be none, as specified in the click option. But after processing it will be an empty list

Reply:
so `container_env_vars` could be `None`? `_process_env_var()` does not seem to handle the `None` situation.

Reply:
Adding a check to prevent iterating over None type

Reply:
Kind of strange our test suites did not catch this, anyway thanks for fixing

Reply:
I think we do have unit tests to test when `container_env_vars` is `None` right?

Reply:
ah actually we don't, we only have a few tests that test on build command

Reply:
Added a unit test for the None case